kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -1,7 +1,23 @@
 resolver kube-dns.kube-system.svc.cluster.local;
 
+{% if nginx_proxy_ssl == "enable" %}
+server {
+    listen 80      default_server;
+    listen [::]:80 default_server;
+    server_name    {{ cluster_address }};
+    return 302     https://$server_name$request_uri;
+
+}
+
+server {
+    listen 443      ssl http2 default_server;
+    listen [::]:443 ssl http2 default_server;
+    include         snippets/nginx-proxy-ssl.conf;
```

Comment:
This is already inside the `nginx-proxy` subdirectory (in the source) or in the `nginx-proxy` image (when built by docker) so you can call this just `ssl.conf`.